app/models/trainee.rb

```diff
@@ -175,4 +175,8 @@ def self.ordered_by_drafts_clause
       END
     SQL
   end
+
+  def training_route
```

Comment:
We could make this private and delegate the methods we need to it. So... you'd end up with `trainee.requires_start_date?`. Would that clean things up a bit more?

Reply:
I was thinking the same, but then you have to list out all the methods which is kind of a pain. I wasn't keen on the idea since if you add/remove methods on `TrainingRouteManager` you have to update the `delegate` list. However, I'm closed to the idea.